src/v/security/audit/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ v_cc_library(
     v::bytes
     v::utils
     v::config
+    v::cluster
```

Comment:
comment: surprised there wasn't a need to update the bazel `BUILD` file here

Reply:
it's already on there https://github.com/redpanda-data/redpanda/blob/ef579f2d4af960f6cf489fe4b6f13256f953ffbe/src/v/security/audit/BUILD#L43

presumably because we already had `cluster::controller` plugged in. so i think this is a case of bazel being more strict.

Reply:
TIL

Reply:
TIF (Tomorrow I (will) Forget)